spread.yaml

```diff
@@ -16,17 +16,31 @@ backends:
                 kernel: GRUB 2
             - debian-unstable-64:
                 kernel: GRUB 2
+            - fedora-25-64
 
 path: /root/spread
 
 restore: |
-    apt autoremove -y
+    case "$SPREAD_SYSTEM" in
+        fedora-*)
+            dnf clean all -y
+            dnf clean metadata
```

Comment:
Ok thanks, perhaps we should report this, according to http://dnf.readthedocs.io/en/latest/command_ref.html#clean-command-label `clean all` should.. well, clean all ;)